### ISSUE TITLE
Possiblity to add other SAT/Servers

### ISSUE THREAD

**Issue by dg7bbp** [ac2cz/Falcon#70]
The current version only  supports FalconSat-3 with PFS-n source callsigns,
I have just migrated my old ka9q DOS patch for sending Pacsat broadcasts to linux. 

EA4GPZ and DB2OS asks in https://forum.amsat-dl.org/index.php?thread/3126-ea4gpz-first-tests-of-a-narrowband-data-modem-for-qo-100/
for a open source server software. But I think we need a modern client software. Can you extend the current software for additional satellites or just changing the possibility for changing the source callsigns. 

I found it is possible to change the callsigns in the FalconSat3.properties file. But switching between satellites seems to be fine feature. Just scan for .properties file and offer a selection of different satellites and a command line parameter to start with given satellite file
 
73 Jens, dg7bbp

**Comment by ac2cz** [comment]
Hi Jens, sorry for the delay in my response. I have not been as focused on the hobby during the Covid-19 lock down.  

I could update the Pacsat ground station to listen to another server on a different spacecraft, yes.
  
With that said, the thread you point to seems to be asking for the server software on the spacecraft.  I don't have that.  I was not part of the development of the BBS itself.  Let me know if that help.

**Comment by ac2cz** [comment]
Jens, did you make progress on this?  I realize it was a while ago.  If there are other changes needed to support QO-100 then I am willing to make them.

**Comment by ac2cz** [comment]
Jens, did you ever progress with the QO-100 project?  Just to let you know that the Ground station will now support Mir-Sat-1 as well as FalconSat-3.  By adding support for a second spacecraft, it would now be very easy to support a third.

**Comment by dg7bbp** [comment]
Chris, nice to hear, that you have added support for other sats.

 I stopped the project for the QO-100 . I first thought to continue with the possiblity to use the modem from dj0abr. After reading the specification I realised, that the modem  allows only payloads of 219 bytes. The modem has an integrated file transfer option. The use of the old pacsat protocol seems not to be necessary for transfering files.
See:
https://wiki.amsat-dl.org/doku.php?id=en:hsmodem:start
https://wiki.amsat-dl.org/doku.php?id=en:hsmodem:packer

73 Jens
